Tests/ClrAssembly/ClrAssembly.csproj

```diff
@@ -20,7 +19,7 @@
       <Output TaskParameter="Path" PropertyName="DotNetFrameworkPath" />
     </GetFrameworkPath>
   </Target>
-  <Target Name="LoadOrder" Inputs="@(LoadOrderCSFile)" Outputs="@(LoadOrderCSFile->'$(OutputPath)\$(Prefix)%(Filename).dll')">
+  <Target Name="LoadOrder" Inputs="@(LoadOrderCSFile)" Outputs="@(LoadOrderCSFile->'$(OutputPath)\$(Prefix)%(Filename).dll')" Condition="'$(Staging)' != 'true'">
```

Comment:
Is there a way to skip the whole project on staging?

Reply:
No, I tried adding it to the Project element and it didn't like it.

Reply:
In actuality, this doesn't work either, the staging parameter is not passed to the build right now,